File: multilibrary/src/main/java/com/zhongjh/albumcamerarecorder/camera/util/LogUtil.java
package com.zhongjh.albumcamerarecorder.camera.util;

import android.util.Log;

import static com.zhongjh.albumcamerarecorder.BuildConfig.DEBUG;


/**
 * 日志工具
 * @author zhongjh
 */
public class LogUtil {

    private static final String DEFAULT_TAG = "ZhongJH";

    public static void i(String tag, String msg) {
        if (DEBUG) {
            Log.i(tag, msg);
        }
    }

    public static void v(String tag, String msg) {
        if (DEBUG) {
            Log.v(tag, msg);
        }
    }

    public static void d(String tag, String msg) {
        if (DEBUG) {
            Log.d(tag, msg);
        }
    }

    public static void e(String tag, String msg) {
        if (DEBUG) {
            Log.e(tag, msg);
        }
    }

    public static void i(String msg) {
        i(DEFAULT_TAG, msg);
    }

    public static void v(String msg) {
        v(DEFAULT_TAG, msg);
    }

    public static void d(String msg) {
        d(DEFAULT_TAG, msg);
    }

    public static void e(String msg) {
        e(DEFAULT_TAG, msg);
    }
}

File: multilibrary/src/main/java/com/zhongjh/multimedia/camera/util/LogUtil.kt
package com.zhongjh.multimedia.camera.util

import android.util.Log
import com.github.chrisbanes.photoview.BuildConfig

/**
 * 日志工具
 *
 * @author zhongjh
 */
object LogUtil {
    private const val DEFAULT_TAG = "zhongjh"

    fun i(tag: String, msg: String) {
        if (BuildConfig.DEBUG) {
            Log.i(tag, msg)
        }
    }

    fun v(tag: String, msg: String) {
        if (BuildConfig.DEBUG) {
            Log.v(tag, msg)
        }
    }

    fun d(tag: String, msg: String) {
        if (BuildConfig.DEBUG) {
            Log.d(tag, msg)
        }
    }

    fun e(tag: String, msg: String) {
        if (BuildConfig.DEBUG) {
            Log.e(tag, msg)
        }
    }

    fun i(msg: String) {
        i(DEFAULT_TAG, msg)
    }

    fun v(msg: String) {
        v(DEFAULT_TAG, msg)
    }

    fun d(msg: String) {
        d(DEFAULT_TAG, msg)
    }

    fun e(msg: String) {
        e(DEFAULT_TAG, msg)
    }
}
